airdrop/airdrop.gno

```diff
@@ -185,3 +189,18 @@ func (a *Airdrop) GetConfig() Config {
 func (a *Airdrop) GetAddress() std.Address {
 	return a.address
 }
+
+// CreateSnapshot creates a new snapshot of the current state
+func (a *Airdrop) CreateSnapshot() uint64 {
+	return a.snapshotManager.CreateSnapshot(a.claimedBitmap, a.token.BalanceOf(std.GetOrigCaller()))
+}
+
+// IsClaimedAtSnapshot checks if a claim was made at a specific snapshot
+func (a *Airdrop) IsClaimedAtSnapshot(snapshotID uint64, claimID uint64) (bool, error) {
```

Comment:
Does whatever function calling this `IsClaimedAtSnapshot` implements error handler ?

Reply:
Yes I think it should. or should I modify it to throw panic?

Reply:
It's up to you. Therefore I'm resolving this conversation.